library/common/config_template.cc

```diff
@@ -268,4 +268,11 @@ stats_flush_interval: {{ stats_flush_interval_seconds }}s
     app_id : {{ app_id }}
     app_version : {{ app_version }}
     os: {{ device_os }}
+# Needed due to warning in https://github.com/envoyproxy/envoy/blob/6eb7e642d33f5a55b63c367188f09819925fca34/source/server/server.cc#L546
```

Comment:
Is there an issue for this or intended to be permanent?

Reply:
Unfortunately from the TODOs in that code the only way to set that configuration is in runtime, and the only (for now) way we can set runtime in envoy mobile is this way